test/unit/vl-map-draw-point-action.test.html

```diff
@@ -11,17 +11,68 @@
+        <vl-map-features-layer id="point-layer">
+          <vl-map-draw-point-action id="draw-point-snap-action" data-vl-default-active data-vl-snapping data-vl-snapping-pixel-tolerance="1000">
+            <vl-map-wfs-layer id="stromendwater" data-vl-name="Stromend waterlichamen" data-vl-url="https://geoserver.vmm.be/geoserver/vmm/wfs" data-vl-layers="owl_l" data-vl-max-resolution="4">
+     		  <vl-map-layer-style data-vl-color="rgba(6, 163, 247, 0.4)" data-vl-border-size="4" data-vl-border-color="rgba(6, 163, 247, 1)"></vl-map-layer-style>
+            </vl-map-wfs-layer>
+            <vl-map-wfs-layer id="stilstaandwater" data-vl-name="Stilstaand waterlichamen" data-vl-url="https://gisservices.inbo.be/arcgis/services/Watervlakken/MapServer/WFSServer?service=wfs" data-vl-layers="Watervlakken" data-vl-max-resolution="4">
+       		  <vl-map-layer-style data-vl-color="rgba(6, 163, 247, 0.4)" data-vl-border-size="4" data-vl-border-color="rgba(6, 163, 247, 1)"></vl-map-layer-style>
+            </vl-map-wfs-layer>
+          </vl-map-draw-point-action>
+        </vl-map-features-layer>
+      </vl-map>
+    </template>
+  </test-fixture>
+
+  <test-fixture id="vl-map-draw-point-action-snapping">
+    <template>
+      <vl-map id="map-with-draw-point-snap-action">
+        <vl-map-features-layer id="point-layer">
+          <vl-map-draw-point-action id="draw-point-snap-action" data-vl-default-active data-vl-snapping>
+          </vl-map-draw-point-action>
+        </vl-map-features-layer>
+      </vl-map>
+    </template>
+  </test-fixture>
+
   <script type="module">
     import {VlMapDrawPointAction} from '../../src/vl-map-draw-point-action.js';
-
+    import {awaitUntil} from '/node_modules/vl-ui-core/dist/vl-core.js';
+    import {VlCompositeVectorLayer, VlCompositeVectorSource} from '/node_modules/vl-mapactions/dist/vl-mapactions.js';
+    
     suite('vl-map-draw-point-action', () => {
       setup((done) => {
-        customElements.whenDefined('vl-map-draw-point-action').then(() => done());
+        Promise.all([
+          customElements.whenDefined('vl-map-features-layer'),
+          customElements.whenDefined('vl-map-wfs-layer'),
+          customElements.whenDefined('vl-map-layer-style'),
+          customElements.whenDefined('vl-map'),
+          customElements.whenDefined('vl-map-draw-point-action'),
+        ]).then(() => done());
       });
-    
+
       test('een punt teken actie is een map actie', () => {
         assert.isTrue(VlMapDrawPointAction.isVlMapAction());
       });
+    
+      test('snapping wordt goed geconfigureerd als er lagen zijn en pixel tolerantie', async () => {
+        const fixtureMetSnapping = fixture('vl-map-draw-point-action-snapping-met-wfs-layers-fixture');
+        const drawPointMetSnappingOpWfsLagenAction = fixtureMetSnapping.querySelector('vl-map-draw-point-action');
+        await awaitUntil(() => drawPointMetSnappingOpWfsLagenAction.action != null);
```

Comment:
Is een map ready hier niet voldoende?

Reply:
Nee want dan luistert zowel de test als de source code op map ready en weet je nog niet wie van beide er eerst komt. 
Dus we moeten in de test wachten op "het effect dat map ready uitlokt op de actie", zijnde het aanmaken van de actie.